source/content/drush-import.md

```diff
@@ -53,8 +62,33 @@ Importing a Drush site archive as we've prepared it above is easy.
 
   At that point the script will poll as the site containers are spun up and the archive is imported. You can wait for that to complete, or cancel out and check back in your Dashboard.
 
+
+</Tab>
+
+<Tab title="Drupal 9 Composer-managed" id="d9">
+
+1. Install [Terminus 3](/terminus/terminus-3-0) if you have a Composer-managed Drupal 9 site with Drush 11.1.1.
+
+1. Install the [Terminus Conversion Tools](https://github.com/pantheon-systems/terminus-conversion-tools-plugin#installation) plugin. 
+
+1. Run the command below to create the site on Pantheon. Change the `site-machine-name` to your machine's name.
```

Comment:
> Change the `site-machine-name` to your machine's name.

This sounds like there is an existing site. Maybe "Change the `site-machine-name` to a unique name that you would like to use for your site."?